charts/argo-workflows/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v3.5.0
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.35.0
+version: 0.35.1
```

Comment:
Since this PR is `feat` , please bump minor version.

Reply:
done